<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactDialog</class>
 <widget class="QDialog" name="ContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="firstNameEdit">
      <property name="placeholderText">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lastNameEdit">
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneEdit">
      <property name="placeholderText">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailEdit">
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="createButton">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ContactDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
